### PR TITLE
Document how to disable YellowBox

### PR DESCRIPTION
**Pull request by grabbou**
As mentioned in #7121 by @javache, this can already be disabled. Might be useful if users upgrade to 0.25 and want to hide warnings about wrong React imported (something they can't really control until community upgrades).

This should be mentioned in the release notes once it's merged.

### CODE REVIEW

File: docs/Debugging.md [hunk 1]
```diff
@@ -31,7 +31,16 @@ Using `console.warn` will display an on-screen log on a yellow background. Click
 
 You can use `console.error` to display a full screen error on a red background.
 
-These boxes only appear when you're running your app in dev mode.
+By default, the warning box is enabled in `__DEV__`. Set the following flag to disable it:
+```js
+console.disableYellowBox = true;
+console.warn('YellowBox is disabled.');
+```
+Warnings can be ignored programmatically by setting the array:
```
Comment (javache) [inline docs/Debugging.md]: "Specific warnings can be ignored..."

**Comment by grabbou** [comment]
@facebook-github-bot shipit

File: docs/Debugging.md (hunk 1 above)
Comment (grabbou) [inline docs/Debugging.md]: Thanks! Shall we update the `YellowBox.js` docs? That was a straight copy

Comment (javache) [inline docs/Debugging.md]: Yeah, we should. Can you send a PR?

**Pull request closed without merging**